test/compile/compile.sh

```diff
@@ -410,23 +409,23 @@ echo "**********************************************************"
 
 wc -l report-test*
 echo
-echo  "Legend:"
-echo ${arr[1]}
-echo ${arr[2]}
-echo ${arr[3]}
-echo ${arr[4]}
-echo ${arr[5]}
-echo ${arr[6]}
-echo ${arr[7]}
-echo ${arr[8]}
-echo ${arr[9]}
-echo ${arr[10]}
-echo ${arr[11]}
-echo ${arr[12]}
-echo ${arr[13]}
-echo ${arr[14]}
-echo ${arr[15]}
-echo ${arr[16]}
-echo ${arr[17]}
-echo ${arr[18]}
-echo ${arr[19]}
+echo "Legend:"
+echo "${arr[1]}"
+echo "${arr[2]}"
+echo "${arr[3]}"
+echo "${arr[4]}"
+echo "${arr[5]}"
+echo "${arr[6]}"
+echo "${arr[7]}"
+echo "${arr[8]}"
+echo "${arr[9]}"
+echo "${arr[10]}"
+echo "${arr[11]}"
+echo "${arr[12]}"
+echo "${arr[13]}"
+echo "${arr[14]}"
+echo "${arr[15]}"
+echo "${arr[16]}"
+echo "${arr[17]}"
+echo "${arr[18]}"
+echo "${arr[19]}"
```

Comment:
We could use a loop here.

Reply:
yes, it can be replaced with `for i in {1..19}; do echo "${arr[$i]}"; done` but I didn't want to make more changes than necessary